src/cli.rs

```diff
@@ -250,8 +259,12 @@ impl Rover {
             // if a request hasn't been made yet, this cell won't be populated yet
             self.client
                 .fill(
-                    get_configured_client(self.accept_invalid_certs, self.accept_invalid_hostnames)
-                        .expect("Could not configure the request client"),
+                    get_configured_client(
```

Comment:
Maybe switching to a builder might come in handy eventually

Reply:
agreed - opened issue #844